content/hacking-atom/sections/hacking-on-atom-core.md

```diff
@@ -41,6 +41,7 @@ There are a couple benefits of running Atom in Dev Mode:
 
 1. When the `ATOM_DEV_RESOURCE_PATH` environment variable is set correctly, Atom is run using the source code from your local `atom/atom` repository. This means that you don't have to run <span class="platform-mac platform-linux">`script/build`</span><span class="platform-windows">`script\build`</span> every time you change code. Just restart Atom 👍
 1. Packages that exist in <span class="platform-mac platform-linux">`~/.atom/dev/packages`</span><span class="platform-windows">`%USERPROFILE%\.atom\dev\packages`</span> are loaded instead of packages of the same name normally loaded from other locations. This means that you can have development versions of packages you use loaded but easily go back to the stable versions by launching without Dev Mode.
+1. Packages that contain stylesheets, such as syntax themes, will have those stylesheets automatically reloaded by the [dev-live-reload](https://github.com/atom/dev-live-reload) package. This does not live reload javascript or coffeescript files, you'll need to reload the window (`window:reload`) to see changes to those.
```

Comment:
I would properly capitalize JavaScript and CoffeeScript.  In addition, would a dash make more sense than a comma here?

Reply:
Sure: 2c025b6